R/req-body.R

```diff
@@ -106,14 +107,30 @@ req_body_json <- function(req, data,
 
 #' @export
 #' @rdname req_body
-#' @param ... <[`dynamic-dots`][rlang::dyn-dots]> Name-data pairs used send
-#'   data in the body. For `req_body_form()`, the values must be strings (or
-#'   things easily coerced to string); for `req_body_multipart()` the values
-#'   must be strings or objects produced by
-#'   [curl::form_file()]/[curl::form_data()].
+req_body_json_modify <- function(req, ...) {
```

Comment:
To avoid accessing the internals like in your example.

Reply:
Is there any way I can modify a nested value?

E.g. how do I modify the value of `b` in `list(a = list(b = 1))`? If I can't do that then I need to somehow extract `a`, then modify `b` and then assign `a`.
There is a reason I used `assign_in()` in my example... This devilish API hides the cursor token in `c("pageCriteria", "identifier", "token")` and this feels like an example of over-engineering or pure evil 😞

Reply:
I think `modifyList()` handles that case, but not `modify_list()`, which isn't really right for modifying json anyway, since it's designed to allow you to set multiple values with the same name. I'll dig into this a bit more.